### PR TITLE
added button

### PR DESCRIPTION
**Pull request by FreedomChuks**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,5 +9,8 @@
 <h1>My first web page and more
 </h1>
 <h2>another</h2>
+<center>
+<Button style="background-color: brown; padding: 15px; color: white;">Click Me</Button>
```
Comment (peatsham) [inline index.html:13]: why did you add this

Comment (FreedomChuks) [reply index.html:13]: it suppose to make it look good
